<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1341</width>
    <height>833</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1361</width>
     <height>841</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/assets/EmployeeBackground.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>./assets/EmployeeBackground.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="DataLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1341</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 26pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Data Visualization</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="CountryRepButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>361</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string> Country Representation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SalaryDisButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>320</y>
     <width>361</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Salary Distribution</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AgeSalButton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>320</y>
     <width>361</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 15pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Age vs Salary</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GoBackButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>650</y>
     <width>541</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 25pt &quot;Segoe UI Black&quot;;</string>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
